tests/TestCase/ORM/Behavior/TranslateBehaviorShadowTableTest.php

```diff
@@ -137,6 +137,18 @@ public function testDefaultPluginAliases(): void
         $this->_testFind('SomeRandomPlugin.Articles');
     }
 
+    public function testComplexLocales(): void
```

Comment:
Duplicating this test method isn't necessary, this class already inherits multiple tests from the test class for Eav strategy.

Reply:
Ah, I wouldn't have thought it extends eav.